### ISSUE TITLE
Product Attribute now showing correctly on product edite page

### ISSUE THREAD

**Issue by mohammed-breky** [woothemes/woocommerce#32511]
### Prerequisites

- [ ] I have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

when clicking on products attribute it's now showing in the correct position

### Expected behavior

it should be showing on the top of list 

### Actual behavior

right now the current view is kind of hidden unless i zoom in and then i only can see few more options

### Steps to reproduce

![fff](https://user-images.githubusercontent.com/37477182/162073911-51bcbb9b-0a69-42f5-ac95-f6403818627f.png)

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://diollo.store
Site address (URL): https://diollo.store
WC Version: 6.3.1
REST API Version: ✔ 6.3.1
WC Blocks Version: ✔ 6.9.0
Action Scheduler Version: ✔ 3.4.0
WC Admin Version: ✔ 3.2.1
Log Directory Writable: ✔
WP Version: 5.9.3
WP Multisite: –
WP Memory Limit: 1 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: LiteSpeed
PHP Version: 8.1.4
PHP Post Max Size: 1 GB
PHP Time Limit: 1500
PHP Max Input Vars: 2000
cURL Version: 7.71.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 10.3.34-MariaDB-log-cll-lve
Max Upload Size: 1 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.3.1
WC Database Prefix: wpto_
Total Database Size: 9.22MB
Database Data Size: 8.65MB
Database Index Size: 0.57MB
wpto_woocommerce_sessions: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_order_items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_order_itemmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_tax_rates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_actionscheduler_actions: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
wpto_actionscheduler_claims: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_actionscheduler_groups: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_actionscheduler_logs: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wpto_commentmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_comments: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_e_events: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_e_submissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_e_submissions_actions_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_e_submissions_values: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_gla_budget_recommendations: Data: 0.11MB + Index: 0.12MB + Engine MyISAM
wpto_gla_merchant_issues: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_gla_shipping_rates: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_gla_shipping_times: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_appointments: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_appointments_excluded: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_appointments_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_post_types: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_reviews: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_review_comments: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_review_guests: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_review_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_review_types: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_smart_filters_indexer: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_jet_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_litespeed_url: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_litespeed_url_file: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_loginizer_logs: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_mailpoet_custom_fields: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_dynamic_segment_filters: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_feature_flags: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_forms: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_mapping_to_external_entities: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_newsletters: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_newsletter_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_newsletter_option: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_newsletter_option_fields: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_mailpoet_newsletter_posts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_newsletter_segment: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_newsletter_templates: Data: 1.04MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_scheduled_tasks: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_mailpoet_scheduled_task_subscribers: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_segments: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_mailpoet_sending_queues: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_settings: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_bounces: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_clicks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_forms: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_newsletters: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_opens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_unsubscribes: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_statistics_woocommerce_purchases: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_stats_notifications: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_subscribers: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_mailpoet_subscriber_custom_field: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_subscriber_ips: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_subscriber_segment: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_user_agents: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_mailpoet_user_flags: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_options: Data: 2.99MB + Index: 0.06MB + Engine MyISAM
wpto_postmeta: Data: 2.86MB + Index: 0.16MB + Engine MyISAM
wpto_posts: Data: 1.57MB + Index: 0.06MB + Engine MyISAM
wpto_termmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_terms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_term_relationships: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_term_taxonomy: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_usermeta: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wpto_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine MyISAM
wpto_wc_admin_note_actions: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
wpto_wc_category_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_customer_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_order_coupon_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_order_product_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_order_stats: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_order_tax_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_product_attributes_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_product_meta_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_wc_rate_limits: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_reserved_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wpto_wc_tax_rate_classes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wpto_wc_webhooks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM

### Post Type Counts ###

attachment: 33
elementor_icons: 1
elementor_library: 3
jet-engine: 3
jet-page-template: 1
jet-theme-core: 1
jet-woo-builder: 7
mailpoet_page: 1
nav_menu_item: 4
page: 11
polylang_mo: 2
post: 3
product: 5
revision: 420
wp_global_styles: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (21) ###

Crocoblock Wizard: by Crocoblock – 1.2.4
Elementor Pro: by Elementor.com – 3.6.4
Elementor: by Elementor.com – 3.6.2
JetBlocks For Elementor: by Crocoblock – 1.3.1
JetBlog For Elementor: by Crocoblock – 2.2.16
JetCompareWishlist For Elementor: by Crocoblock – 1.4.5
JetElements For Elementor: by Crocoblock – 2.6.3
JetEngine: by Crocoblock – 2.11.6
JetMenu: by Crocoblock – 2.1.7
JetPopup: by Crocoblock – 1.5.6
JetReviews For Elementor: by Crocoblock – 2.2.6
JetSearch For Elementor: by Crocoblock – 2.1.16
JetSmartFilters: by Crocoblock – 2.3.11
JetTabs For Elementor: by Crocoblock – 2.1.16
JetThemeCore: by Crocoblock – 2.0.0
JetTricks: by Crocoblock – 1.4.1
JetWooBuilder For Elementor: by Crocoblock – 1.12.2
JetProductGallery For Elementor: by Crocoblock – 2.1.1
Loginizer: by Softaculous – 1.7.0
MailPoet 3 (New): by MailPoet – 3.86.0
WooCommerce: by Automattic – 6.3.1

### Inactive Plugins (4) ###

Jet Appointments Booking: by Crocoblock – 1.6.9
JetBooking: by Crocoblock – 2.4.3
LiteSpeed Cache: by LiteSpeed Technologies – 4.6
W3 Total Cache: by BoldGrid – 2.2.1

### Dropin Plugins (1) ###

db.php: db.php

### Settings ###

API Enabled: –
Force SSL: –
Currency: LYD (ل.د)
Currency Position: left_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 3
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #15 - /shop/
Cart: #16 - /cart/
Checkout: #17 - /checkout/
My account: #18 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Hello Elementor
Version: 2.5.0
Author URL: https://elementor.com/?utm_source=wp-themes&utm_campaign=author-uri&utm_medium=wp-dash
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 69
Oldest: 2022-04-01 15:39:59 +0200
Newest: 2022-04-06 22:19:28 +0200

Pending: 1
Oldest: 2022-04-07 22:19:28 +0200
Newest: 2022-04-07 22:19:28 +0200

### Status report information ###

Generated at: 2022-04-06 23:26:16 +02:00
`

### Isolating the problem

- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Comment by woocommercebot** [comment]
We are adding the `status: needs reproduction` label to this issue to try reproduce it on the   current released version of WooCommerce.

  Thank you for your patience.

**Comment by joey1613** [comment]
How did you solve this issue? I've got the exact same issue for my "attributes" and "variations" section on the "edit product page"

**Comment by mohammed-breky** [comment]
> How did you solve this issue? I've got the exact same issue for my "attributes" and "variations" section on the "edit product page"

unfortunately not yet, tried to reinstall woocommerce but it didn't solve the issue, also the same issue is in attribute and variations tabs

**Comment by swatipawarGS** [comment]
Hi @mohammed-breky,

Thank you for taking the time to report this bug, we really appreciate your help. We are unable to reproduce the issue on our end using WooCommerce 6.3.1 version and WordPress 5.9.3 version. 

Please find below screenshots for reference :

Attributes : 
![#32511(att)](https://user-images.githubusercontent.com/94531721/162386310-c10edc78-fdae-4bbd-828e-549ad555db85.png)

variations : 
![#32511(var)](https://user-images.githubusercontent.com/94531721/162386350-22591a3a-4a6b-469d-b4b0-7f5e1f4e63f2.png)

Please provide us more detailed and any additional steps required to repro this issue which may help us to evaluate it further.

**Comment by mohammed-breky** [comment]
i tried different versions of woocommerce but the exact same issue appears every time 
i aslo tried woocommerce official method of uninstalling using define( 'WC_REMOVE_ALL_DATA', true );
after that i get this error :
Deletion failed: There has been a critical error on this website.Learn more about troubleshooting WordPress.

**Comment by mohammed-breky** [comment]
i deleted the whole wordpress installation and reinstall woocommerce, it works now.

**Comment by mohammed-breky** [comment]
i deleted the whole wordpress installation and reinstall woocommerce, it works now

**Comment by joey1613** [comment]
For me it was an issue with "JetThemeCore" plugin created by crocoblock.com, they said they are aware of the issue and working on it and I have temporarily rolled back the plugin version.

**Comment by mohammed-breky** [comment]
> For me it was an issue with "JetThemeCore" plugin created by crocoblock.com, they said they are aware of the issue and working on it and I have temporarily rolled back the plugin version.

can you please link the thread of crocoblock

**Comment by joey1613** [comment]
there is no current thread, I just spoke with support

**Comment by mohammed-breky** [comment]
can you forward the email with the exact plugin version to me ?

**Comment by joey1613** [comment]
The issue is with the latest version, JetThemeCore 2.0.0 released on March 28, 2022. Like I mentioned earlier, the issue goes away if you roll it back one version. 

You can keep an eye on the link below for when they release the next version.

https://crocoblock.com/changelog/

**Comment by mohammed-breky** [comment]
ok, thank you for addressing the issue

**Comment by lovefist43** [comment]
I have found a temporary fix. While adding attributes to product data, run a video on youtube to memorize the locations of boxes and buttons. Then use inspect element to give left margins to buttons so they can be visible in the center and then click the buttons in the inspect element mode. This way I was able to add attributes and variations. I hope the plugin team comes up with the update soon.

note: Before doing that I also went to jetcore plugin settings and enabled the products attribute (e.g color) option. I'm not sure if that helped but after that, the input field did appear although in a distorted position that required margins.
